### PR TITLE
Clarify docs about CA fingerprints and Fleet

### PR DESCRIPTION
**Pull request by dedemorton**
This change was recommended by @joshdover [here](https://github.com/elastic/observability-docs/pull/1555#discussion_r808041585).

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
Wording LGTM with a minor suggestion.

I'd like to get a review from @elastic/es-security before merging this in.

File: docs/reference/setup/install/connect-clients.asciidoc
Comment (jkakavas) [inline docs/reference/setup/install/connect-clients.asciidoc:10]:
```suggestion
{agent}s are automatically configured to trust the CA certificate.
```

The CA certificate is what is ultimately trusted. Whether the actual CA certificate  or the fingerprint of said CA certificate is used to do that, is more of an internal implementation detail for us. I think that expressing this as "trusting the certificate" matches terminology that users are used to read and is the least probable to raise any eyebrows.

File: docs/reference/setup/install/connect-clients.asciidoc
Comment (jkakavas) [inline docs/reference/setup/install/connect-clients.asciidoc:15]: I know this is not part of this change but we can also mention the path to the CA certificate  in these instructions which is always : `$ES_HOME/config/certs/http_ca.crt` and later on in this file, maybe in the top after the " You can also copy the CA certificate to your machine and configure your client to use it. " section ? 

Also, we can remove the 
`<timestamp>:: The timestamp of when the auto-configuration process created the security files directory.` callout as it is a remnant of a previous iteration.

**Review by jkakavas — COMMENTED** [review]
I suggested a small change

File: docs/reference/setup/install/connect-clients.asciidoc
Comment (jrodewig) [reply docs/reference/setup/install/connect-clients.asciidoc:15]: @dedemorton Let me know how you want to handle this. I'm happy to do one of the following:

A) Perform this work in a separate PR once this is merged in.
B) Push these changes to your branch.
C) Let you make the changes on this branch.

I mostly don't want to block you or push stuff to your PR without your permission.

**Review by joshdover — APPROVED** [review]
Thanks for follow up on this!

File: docs/reference/setup/install/connect-clients.asciidoc
Comment (dedemorton) [inline docs/reference/setup/install/connect-clients.asciidoc:5]:
```suggestion
`$ES_HOME/config/certs/http_ca.crt`. The hex-coded SHA-256 fingerprint of this
```

File: docs/reference/setup/install/connect-clients.asciidoc
Comment (dedemorton) [inline docs/reference/setup/install/connect-clients.asciidoc:3]: @jkakavas Can you take another look? I thought it made sense to mention the file path where we talk about the file being stored on disk. 

Also...is hex-coded correct? Or should this be hex-encoded?

Comment (jkakavas) [reply docs/reference/setup/install/connect-clients.asciidoc:3]: Good catch ! It’s hex-encoded

**Review by jkakavas — APPROVED** [review]
LGTM thank you 🙏

File: docs/reference/setup/install/connect-clients.asciidoc
Comment (dedemorton) [inline docs/reference/setup/install/connect-clients.asciidoc:5]:
```suggestion
`$ES_HOME/config/certs/http_ca.crt`. The hex-encoded SHA-256 fingerprint of this
```

**Pull request merged**
